src/forms/labels/labeled-field.js

```diff
@@ -54,23 +66,28 @@ const defaultProps = {
   hideErrorLabel: false,
 }
 
+const DefaultLabelComponent = InputLabel
+const DefaultErrorComponent = InputError
```

Comment:
I don't know if renaming these as separate vars is adding a lot. But if you want to keep them separate, you should use constant naming: `DEFAULT_LABEL_COMPONENT`